### PR TITLE
Implement chat into new transaction structure

### PR DESCRIPTION
**Pull request by andyttran**

### CODE REVIEW

File: snapmarketapp/app/scripts/transaction/transaction-controllers.js [hunk 1]
```diff
@@ -84,11 +76,121 @@ angular.module('transaction.controllers', [])
...
+    url: "#/tab/transaction/buyOffers"
+  };
+
+  $scope.handle = Db.getAuth().uid;
+  $scope.showTime = false;
+
+
+  function scrollBottom() {
+    $timeout(function() {
+      var startHandle = _.find($ionicScrollDelegate._instances, function (s) {
```
Comment (vtumrukota) [inline snapmarketapp/app/scripts/transaction/transaction-controllers.js]: Lets be more clear about the s variable and maybe throw in a comment about what its doing

**Comment by vtumrukota** [comment]
Looks good overall, lets just get the ionic.project removed (clear git cache)

**Comment by josephlin55555** [comment]
i concur

**Comment by dgrundfest** [comment]
Looks good to me. Merging in

**Pull request merged**